docs/dev_guide/documentation.rst

```diff
@@ -33,18 +33,19 @@ Usage
 #####
 
 All of the SunPy documentation is contained in the ``docs`` folder and
-code comments. The examples from the example gallery can be found in
-``examples``. To build the documentation locally you must have Sphinx
-(as well as Numpydoc, astropy-helpers, and sphinx-gallery) installed on
+code docstings/comments. The examples from the example gallery can be found in
+the ``examples`` folder. To build the documentation locally you must have
+Sphinx (as well as Numpydoc, astropy-helpers, and sphinx-gallery) installed on
 your computer. In the root directory run ::
 
     python setup.py build_docs
 
-This will generate HTML documentation for SunPy in the ``docs/_build/html``
+This will generate HTML documentation for SunPy in the ``docs/docs/_build/html``
```

Comment:
The docs/docs path is a upstream bug that needs to be fixed.

Reply:
Sounds good, I reverted the change